### ISSUE TITLE
When aux loss is enabled, plot diffusion loss in a separate window in tensorboard

### ISSUE THREAD

**Issue by kabachuha** [bghira/SimpleTuner#2188]
Hello!

I'm testing the new regularization loss capabilities, however it's very confusing when the tensorboard plots only the auxiliary loss itself and the total loss (which is often negative). The diffusion loss (mse/huber/etc.) is overshadowed and it's hard to grasp the actual training dynamics.

Could you add a new visible section for the diffusion loss in the reporting mechanisms?

Thank you!

**Comment by kabachuha** [comment]
Btw, pure layersync seems not to be really great. For me, it caused this sepia tint when I trained a lora, correlated with rapid increase in layer similarity, so not recommending it. I think it's intended for pre-training and not changing the representations on the fly.

On the other hand, REPA/CREPA are using a MLP, so the features are not "raw" tampered with, and they should be better for fine-tuning/lora.

<img width="884" height="446" alt="Image" src="https://github.com/user-attachments/assets/0d6116ec-4769-463d-974f-07c631a3be70" />

https://github.com/user-attachments/assets/6cab0c59-aca5-4e33-ad62-96f14e3ed508

Looking forward for the CREPA fix! (for the continued CREPA issue, see #2173)

**Comment by bghira** [comment]
i'm not sure what you're requesting here. can you elaborate further?

**Comment by kabachuha** [comment]
If you mean the first point, there are currently two groups of loss showing rentangles in tensorboard (or other reporting mechanisms): aux loss, train loss and optimization loss. The last two are dependent on aux loss and the progression of the usual diffusion/flow matching Gaussian loss is hard to see because they both sum for the display. What I suggest is to show a pure diffusion/reconstruction loss window when there is aux loss enabled, so one can track how the model handles the actual images content

The second was about the weirdness with layer sync. From the test it seems it can wash out the representations when the model is not trained from scratch, but this is not much explored yet.

I saw your new fix for CREPA, I'll test it as soon as possible
